packages/schedule/src/useSchedule.ts

```diff
@@ -7,14 +7,24 @@
 
 import { useState, useLayoutEffect } from 'react';
 
-export default function useSchedule({ duration = 1250, delayMS = 750, loop = true } = {}) {
+export interface IUseScheduleReturnValue {
+  elapsed: number;
+  delayMS: number;
+  delayComplete: boolean;
+}
+
+export const useSchedule = ({
+  duration = 1250,
+  delayMS = 750,
+  loop = true
+} = {}): IUseScheduleReturnValue => {
```

Comment:
This should define an interface to share its prop structure with `ScheduleContainer`. Possibly something like:

```ts
interface IUseScheduleProps {
  duration?: number;
  delayMS?: number;
  loop?: boolean;
}
```

Reply:
Oh yeah, didn't do this because those arguments were inferred. But I can see why having `IUseScheduleProps` is better. I'll go ahead and create this interface, thanks!

Reply:
Updated, thanks for the suggestion!